server/service/src/settings.rs

```diff
@@ -7,6 +8,7 @@ pub struct Settings {
     pub server: ServerSettings,
     pub database: DatabaseSettings,
     pub sync: Option<SyncSettings>,
+    pub logging: LoggingSettings,
```

Comment:
Can it be optional Option<LoggingSettings>? i.e. if not specified use the defaults? (don't break existing setups)

Reply:
yeah, I wondered about that. can do - I was thinking that it was nice to have it specified in the config, so that the current logging options were plain to see.